consul-core/src/main/java/com/smoketurner/dropwizard/consul/core/ConsulAdvertiser.java

```diff
@@ -162,4 +173,18 @@ public void deregister() {
             LOGGER.error("Failed to deregister service from Consul", e);
         }
     }
+
+    private String buildHealthCheckUrl(int adminPort) {
+        StringBuilder healthCheckUrl = new StringBuilder("http://");
```

Comment:
Instead of building a string here, is it possible to create a `Uri` instance then call its `.toString()` method instead?